phost/src/chain_client.rs

```diff
@@ -175,6 +175,22 @@ pub fn storage_value_key_vec(module: &str, storage_key_name: &str) -> Vec<u8> {
     key
 }
 
+fn storage_map_key_vec(module: &str, storage_item: &str, storage_item_key: &str) -> Vec<u8> {
```

Comment:
// Calculates the Substrate storage key prefix for a StorageMap

Reply:
Done